cognite/client/utils/_experimental.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+import warnings
+from typing import Literal
+
+
+class FeaturePreviewWarning(FutureWarning):
+    def __init__(self, api_version: Literal["alpha", "beta"], sdk_version: Literal["alpha", "beta"], feature_name: str):
+        self.api_version = api_version
```

Comment:
Maybe api_maturity and sdk_maturity instead?

Reply:
Agree, that is a better word.